tools/old/stacksnoop.py

```diff
@@ -119,10 +119,7 @@
     (task, pid, cpu, flags, ts, msg) = b.trace_fields()
     if msg != "":
         (reg, addr) = msg.split(" ")
-        if offset:
-            ip = b.ksymaddr(int(addr, 16))
-        else:
-            ip = b.ksym(int(addr, 16))
+        ip = b.ksym(int(addr, 16), show_address=offset)
```

Comment:
Whoops, disregard this and the following ones. I misinterpreted `offset` as a string value.